README.md

```diff
@@ -55,7 +56,7 @@ packages.
 
  * Save and exit.
 
- * Run `make`
+ * Run `make` (for a real benchmarking setup do *not* use `-j`)
```

Comment:
I don't think it's worth saying "don't do this" since it's not something people would do unless we told them to. If we enumerate all the "don't do this" possibilities, we'll end up like this http://dilbert.com/strip/2017-08-24 ;)